CommonCompilerConfig.cmake

```diff
@@ -75,12 +77,19 @@ endif()
 # Init exported properties
 set(COMMON_COMPILER_DEFINITIONS)
 set(COMMON_COMPILER_DEFINITIONS_STR)
-set(COMMON_C_FLAGS_OPTIMIZED)
-set(COMMON_C_FLAGS_UNOPTIMIZED)
-set(COMMON_CXX_FLAGS_OPTIMIZED)
-set(COMMON_CXX_FLAGS_UNOPTIMIZED)
-set(COMMON_LINKER_FLAGS_OPTIMIZED)
-set(COMMON_LINKER_FLAGS_UNOPTIMIZED)
+
+set(INTERNAL_CXX_FLAGS_OPTIMIZED)
+set(INTERNAL_LINKER_FLAGS_OPTIMIZED)
+
+
```

Comment:
After re-examining my state of mind at the time of these changes, the double blank line may have been intentional, serving as separation between the INTERNAL and EXTERNAL flag sets. :smiley: